Makefile

```diff
@@ -93,43 +93,21 @@ endef
 .PHONY: config-debug
 config-debug:
 	@echo "## xNVMe: make config-debug"
-	CC=$(CC) CXX=$(CXX) $(MESON) setup $(BUILD_DIR) --buildtype=debug
-	@echo "## xNVMe: make config-debug [DONE]"
-
-define config-uring-help
-# Configure Meson to compile xNVMe without: SPDK, and libvfn
-endef
-.PHONY: config-uring
-config-uring:
-	@echo "## xNVMe: config-uring"
 	CC=$(CC) CXX=$(CXX) $(MESON) setup $(BUILD_DIR) \
-	   -Dwith-spdk=disabled \
-	   -Dwith-liburing=enabled \
-	   -Dwith-libvfn=disabled
-	@echo "## xNVMe: config-uring [DONE]"
-
-define config-libvfn-help
-# Configure Meson to compile xNVMe without: SPDK, and liburing
-endef
-.PHONY: config-libvfn
-config-libvfn:
-	@echo "## xNVMe: config-libvfn"
-	CC=$(CC) CXX=$(CXX) $(MESON) setup $(BUILD_DIR) \
-	   -Dwith-spdk=disabled \
-	   -Dwith-liburing=disabled \
-	   -Dwith-libvfn=enabled
-	@echo "## xNVMe: config-uring [DONE]"
+		 --buildtype=debug
+	@echo "## xNVMe: make config-debug [DONE]"
 
 define config-slim-help
-# Configure Meson to compile xNVMe without: SPDK, liburing, and libvfn
+# Configure Meson to compile xNVMe without any third-part libraries
 endef
 .PHONY: config-slim
 config-slim:
 	@echo "## xNVMe: make config-slim"
 	CC=$(CC) CXX=$(CXX) $(MESON) setup $(BUILD_DIR) \
-	   -Dwith-spdk=disabled \
-	   -Dwith-liburing=disabled \
-	   -Dwith-libvfn=disabled
+	   -Dwith-spdk=false \
```

Comment:
Having SPDK as a boolean and the other dependencies as features, are intentional. Changing the make-helper for SPDK was a typo.

When the option is a feature, then it has a third value-option (beyond true/false, enabled/disabled). The "auto" state, which works neatly with the meson ``dependency``, e.g. ``dependency(..., required: get_option('with-libaio')``. When doing so, then by default, then the libraries that xNVMe optionally depend are enabled when found on the system, and disabled otherwise.

It is not possible to use the "auto" feature since it SPDK is provided via subproject and consumed in a un-conventional way. It is on by default, and expects the subproject to take care of it. Unless the user explicitly says ``-Dwith-spdk=false``.